ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/event/ng/EditorFileStatusNotificationOperation.java

```diff
@@ -48,22 +49,26 @@
     private NotificationManager notificationManager;
 
     @Inject
-    public EditorFileStatusNotificationHandler(EventBus eventBus,
-                                               DeletedFilesController deletedFilesController,
-                                               AppContext appContext) {
-        super(VfsFileStatusUpdateDto.class, Void.class);
+    public EditorFileStatusNotificationOperation(EventBus eventBus, DeletedFilesController deletedFilesController, AppContext appContext) {
         this.eventBus = eventBus;
         this.deletedFilesController = deletedFilesController;
         this.appContext = appContext;
+    }
 
+    @Inject
+    public void configureHandler(RequestHandlerConfigurator configurator) {
```

Comment:
The main idea is that smb might initialize class fields by mistake and broke smth while we have general practice to use construction injection.